### PR TITLE
Change label value used during self-registration

### PR DESCRIPTION
**Pull request by krasish**
**Description**
As it is implemented at the moment, CMP uses the distinguishing label's value for input during the self-registration preparation process. However this value might include unsupported characters. This PR changes the input to use the runtime ID.

Changes proposed in this pull request:
- Change clone suffix to runtime id

**Pull Request status**

- [X] Implementation
- [x] Unit tests
- [ ] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: components/director/internal/domain/runtime/service_test.go [hunk 1]
```diff
@@ -192,9 +192,7 @@ func TestService_CreateWithMandatoryLabels(t *testing.T) {
 				return tenantSvc
 			},
 			UIDServiceFn: func() *automock.UidService {
-				svc := &automock.UidService{}
-				svc.On("Generate").Return(runtimeID)
-				return svc
+				return &automock.UidService{}
```
Comment (la4ezar) [inline components/director/internal/domain/runtime/service_test.go:195]: You can extract 
`func() *automock.UidService {
				return &automock.UidService{}
}`
in separate function named 'unusedUIDService' for example.

**Comment by PetarTodorovv** [comment]
Looks okay to me ✅ Only need adaptation in the e2e tests because we add the "distinguish label" in the runtime input and after that validate/assert the expected result. But now the "distinguish label" contains runtimeID which we'll have after registration(we don't have it beforehand when build runtime input)

File: components/director/internal/domain/runtime/resolver.go [hunk 2]
```diff
@@ -221,7 +224,7 @@ func (r *Resolver) RegisterRuntime(ctx context.Context, in graphql.RuntimeInput)
 
 	ctx = persistence.SaveToContext(ctx, tx)
 
-	id, err := r.runtimeService.CreateWithMandatoryLabels(ctx, convertedIn, labels)
+	err = r.runtimeService.CreateWithMandatoryLabels(ctx, convertedIn, id, labels)
```
Comment (la4ezar) [inline components/director/internal/domain/runtime/resolver.go:227]: if err = ...; err != nil {...}

**Review by PetarTodorovv — APPROVED** [review]

**Pull request merged**